src/style.css

```diff
@@ -1 +1,72 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.box {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+
+  position: relative;
+  display: flex;
+  align-items: center;
+  white-space: nowrap;
+
+  text-decoration: none;
+  color: #000;
+  margin-right: 17px;
```

Comment:
![image](https://user-images.githubusercontent.com/38065883/174638222-9b8a1084-83be-4222-a96f-3cbac11d0dd0.png)
None link has this margin, remove this line